src/modules/GroupChannel/components/SuggestedReplies/index.scss

```diff
@@ -5,25 +5,51 @@
 @import '../../../../styles/variables';
 
 .sendbird-suggested-replies {
+  position: relative;
+  display: flex;
+  margin-top: 8px;
+  &.vertical {
+    justify-content: flex-end;
+    align-items: flex-end;
+    flex-wrap: wrap;
+    flex-direction: column;
+    .sendbird-suggested-replies__option:not(:first-child) {
+      margin-top: 8px;
+    }
+  }
+
+  &.horizontal {
+    display: flex;
+    flex-direction: row;
+    overflow-x: scroll;
+
+    margin-left: -$messages-padding;
+    margin-right: -$messages-padding;
+    padding-left: $messages-padding;
+    padding-right: $messages-padding;
+    @include mobile() {
+      margin-left: -$messages-padding-mobile;
+      margin-right: -$messages-padding-mobile;
+      padding-left: calc(40px + $messages-padding-mobile);
+      padding-right: $messages-padding-mobile;
+    }
+    .sendbird-suggested-replies__option:not(:last-child) {
+      margin-right: 8px;
+    }
```

Comment:
Could you please check if the `padding-left` style is correct on both mobile and desktop?

Reply:
Thanks i tested and fixed replies item padding to original

![Screenshot 2024-04-30 at 2 49 19 PM](https://github.com/sendbird/sendbird-uikit-react/assets/16806397/2ca34a54-f277-4a59-925b-17f364b8019a)
![Screenshot 2024-04-30 at 2 49 07 PM](https://github.com/sendbird/sendbird-uikit-react/assets/16806397/0c620f00-8075-481d-993d-6fc5f8e077f7)
![Screenshot 2024-04-30 at 2 48 56 PM](https://github.com/sendbird/sendbird-uikit-react/assets/16806397/e1bb5ecb-5369-4c0f-b101-aeeecbb95e5b)
![Screenshot 2024-04-30 at 2 48 52 PM](https://github.com/sendbird/sendbird-uikit-react/assets/16806397/74714962-43e3-4abe-940f-9413080bd104)
![Screenshot 2024-04-30 at 2 48 10 PM](https://github.com/sendbird/sendbird-uikit-react/assets/16806397/fb104b5d-2249-4507-92cf-9476da2fec56)
![Screenshot 2024-04-30 at 2 47 52 PM](https://github.com/sendbird/sendbird-uikit-react/assets/16806397/70169e6e-d825-4fae-b8a5-b1c112a97eb1)

Reply:
thanks for confirming!